src/main/java/com/dsrm/dsrmbackend/mappers/RoomMapper.java

```diff
@@ -7,7 +7,7 @@
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
 
-@Mapper(componentModel = "spring")
+@Mapper(componentModel = "spring", uses = UserMapper.class)
```

Comment:
Do not use it that way. Please use getReferenceById to set a User in room like with roomType